src/simulators/tensor_network/matrix_product_state.hpp

```diff
@@ -191,8 +193,23 @@ class MPS{
     return 0;
   }
   
+  //----------------------------------------------------------------
+  // function name: update_qubit_and_neighbor
+  // Description: This function is used by apply_measure to propagate
+  // the effects of measurement from the measured qubit to all other
+  // qubits
+  //----------------------------------------------------------------
+  void update_qubit_and_neighbor(uint_t index, 
```

Comment:
I can't see where this function is being used

Reply:
you are right. I removed this function.